src/tailwind.config.js

```diff
@@ -90,6 +91,14 @@ module.exports = {
       },
       ringWidth: {
         '1': '1px',
+      },
+      borderRadius: {
+        'surface': `var(--radius-surface, ${defaultBorders.radiuses['radius-surface']})`,
+        'button': `var(--radius-button, ${defaultBorders.radiuses['radius-button']})`,
+        'input': `var(--radius-input, ${defaultBorders.radiuses['radius-input']})`
+      },
+      borderWidth: {
+        'secondary': `var(--width-secondary, ${defaultBorders.width['width-secondary']})`
```

Comment:
as we discussed offline: `--border-width-secondary` instead of `--width-secondary` please